tdrs-backend/tdpservice/data_files/models.py

```diff
@@ -157,6 +157,14 @@ def filename(self):
         """Return the correct filename for this data file."""
         return self.stt.filenames.get(self.section, None)
 
+    @property
+    def s3_location(self):
```

Comment:
normally s3 location refers to the bucket name, however here I believe we are referring to OBJECT name

Reply:
It resolves to the fqp of the object as can be seen in the `get_s3_upload_path`. I could strip the actual files name off the property if that is more in line with the objects location.

Reply:
In that case it should be ok